compiler/rustc_mir_transform/src/coverage/query.rs

```diff
@@ -1,6 +1,5 @@
 use super::*;
 
-use rustc_middle::mir::coverage::*;
```

Comment:
delete it because it resolved at https://github.com/rust-lang/rust/blob/master/compiler/rustc_mir_transform/src/coverage/mod.rs#L22